prime-dotnet-webapi/Models/Enrollee.cs

```diff
@@ -126,6 +126,8 @@ public Enrollee()
 
         public EnrolleeLinkedEnrolment PaperToEnrolleeLink { get; set; }
 
+        public DateTimeOffset? SelfDeclarationCompleteDate { get; set; }
```

Comment:
Should that be `SelfDeclarationCompletedDate` like we have `Agreement.AcceptedDate`?

Reply:
No, enrollee can answer the questions on one day and accept the TOA on a different date. Especially, for manual adjudication ones.

Reply:
What I mean is past tense "Completed" rather than "Complete"

Reply:
Good point. It is updated.